src/git_changelog/build.py

```diff
@@ -98,7 +98,6 @@ def __init__(
         self.url: str = url
         self.compare_url: str = compare_url
         self.previous_version: Version | None = None
-        self.next_version: Version | None = None
```

Comment:
For backward-compatibility, I'd like to retain this attribute. Hopefully it's easy to retain it: when we do

```python
version.previous_version = self.versions_dict.get(previous_version)
```

...we can just also do

```python
version.previous_version.next_version = version
```

Or something like that :smile:

Reply:
Sure, I re-added the "next_version".
If a version has two or more next versions, the oldest one is assigned (due to iterating through the version list from newest to oldest)

Reply:
Perfect, thanks!